### DISCUSSION TITLE
[Chapter 9] 마왕성 코드 문제

### DISCUSSION THREAD

**Discussion by firewoody237** [cowaine/is-my-code-that-strange#85] [Chapter 09-10 (악마, 이름 설계)] [answered]
이번주는 딱히 질문할 거리가 없어서...
퀴즈용도로 악마들이 많은 엉망진창 마왕성 코드를 만들었습니다.

`com.cowaine.firewoody237.fifth_week.bad_code.multiple_calculator` 패키지 아래 온갖 악행을 저질러놨습니다.
대부분 `Chapter 9`에서 나오는 내용을 생각하며 만들었기에,
나중에 심심하실 때 복습할겸 각자 레포지토리에서 좋은 코드로 바꿔보시기 바랍니다.

부디 복습에 도움이 되었으면 좋겠네요 ㅎㅎ

(`Merge`가 안되어있으면 코드가 안보일 수도 있습니다.)

**Comment by corock** [comment, marked as answer]
PR 병합 후 링크 댓글로 공유해 드리겠습니다.

**Reply by corock** [reply]
[마왕성 코드](https://github.com/cowaine/is-my-code-that-strange/commit/a6d67ba85b50f384332d743b6f2d613a2ba0ddc6) 바로가기
